### DISCUSSION TITLE
WZA - informacja o obsługiwanych typach w zasobie /parcel-management/delivery-services

### DISCUSSION THREAD

**Discussion by bartekc7** [allegro/allegro-api#6667] [Suggestions - resolved]
Pomocna byłaby informacja o możliwym typie paczki do użycia w danej usłudze. "PACKAGE" "DOX" "PALLET"
Analogicznie jak jest z usługami dodatkowymi "additionalServices"
Chodzi o to że wystawiając przesyłki przez API WZA nie wiemy np. czy wybrana usługa umożliwia wysyłkę palet. 

![image](https://user-images.githubusercontent.com/207927/216345915-519ab247-044d-441e-996d-8ca3223f6f31.png)

**Comment by MartaNowaczyk** [comment]
Sugestię wdrożono wraz z nowym API dla WZA - [GET /shipment-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getDeliveryServices),
